app.js

```diff
@@ -28,6 +28,8 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 const index = require('./routes/index');
 app.use('/', index);
 
+const auth = require('./routes/auth.routes');
+app.use('/', auth);
 // ❗ To handle errors. Routes that don't exist or errors that you handle in specific routes
 require('./error-handling')(app);
 
```

Comment:
On line 19 you are requiring the config file, but since you created a new file inside the folder config you need to require the new file as well (similar to the routes when you create a new file for routes you need to require that file here on app.js):
`require('./config/session.config')(app);`
so you can just add this line of code at line 20